openshift/patroni.dc.yml

```diff
@@ -0,0 +1,201 @@
+                valueFrom:
+                  fieldRef:
+                    apiVersion: v1
+                    fieldPath: metadata.namespace
+              - name: PATRONI_KUBERNETES_LABELS
+                value: '{"cluster-name": "${APP_NAME}-patroni", "app.kubernetes.io/name": "patroni"}'
+              - name: PATRONI_SUPERUSER_USERNAME
+                valueFrom:
+                  secretKeyRef:
+                    key: superuser-username
+                    name: ${APP_NAME}-patroni
+              - name: PATRONI_SUPERUSER_PASSWORD
+                valueFrom:
+                  secretKeyRef:
+                    key: superuser-password
+                    name: ${APP_NAME}-patroni
+              - name: PATRONI_REPLICATION_USERNAME
+                valueFrom:
+                  secretKeyRef:
+                    key: replication-username
+                    name: ${APP_NAME}-patroni
+              - name: PATRONI_REPLICATION_PASSWORD
+                valueFrom:
+                  secretKeyRef:
+                    key: replication-password
+                    name: ${APP_NAME}-patroni
+              - name: APP_USER
+                valueFrom:
+                  secretKeyRef:
+                    key: db-username
+                    name: ${APP_NAME}-patroni
+              - name: APP_PASSWORD
+                valueFrom:
+                  secretKeyRef:
+                    key: db-password
+                    name: ${APP_NAME}-patroni
+              - name: APP_DATABASE
+                valueFrom:
+                  secretKeyRef:
+                    key: db-name
+                    name: ${APP_NAME}-patroni
+              - name: PATRONI_SCOPE
+                value: ${APP_NAME}-patroni
+              - name: PATRONI_NAME
+                valueFrom:
+                  fieldRef:
+                    apiVersion: v1
+                    fieldPath: metadata.name
+              - name: PATRONI_LOG_LEVEL
+                value: WARNING
+              - name: PATRONI_POSTGRESQL_DATA_DIR
+                value: /home/postgres/pgdata/pgroot/data
+              - name: PATRONI_POSTGRESQL_PGPASS
+                value: /tmp/pgpass
+              - name: PATRONI_POSTGRESQL_LISTEN
+                value: 0.0.0.0:5432
+              - name: PATRONI_RESTAPI_LISTEN
+                value: 0.0.0.0:8008
+              image: docker-registry.default.svc:5000/${IMAGE_NAMESPACE}/${APP_NAME}-patroni:${IMAGE_TAG}
+              imagePullPolicy: Always
+              name: ${APP_NAME}-patroni
+              ports:
+                - containerPort: 8008
+                  protocol: TCP
+                - containerPort: 5432
+                  protocol: TCP
+              resources:
+                requests:
+                  cpu: ${CPU_REQUEST}
+                  memory: ${MEMORY_REQUEST}
+                limits:
+                  cpu: ${CPU_LIMIT}
+                  memory: ${MEMORY_LIMIT}
+              terminationMessagePath: /dev/termination-log
```

Comment:
so we do already have some logging going to file :) should we follow this path paradigm for other logging for the time being?

Reply:
Have not even looked into this, but it appears as if it's got some basic logging baked in. As it stands, `/dev/termination_log` is empty.